packages/api/schema/stryker-core.json

```diff
@@ -133,6 +133,18 @@
         }
       }
     },
+    "jsonReporterOptions": {
+      "title": "JsonReporterOptions",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+        "baseDir": {
+          "description": "The output folder for the json report.",
+          "type": "string",
+          "default": "reports/mutation/json"
```

Comment:
Since the JSON report will be a single file, I think it makes sense to default to `reports/mutation` here. So